doc/how_to_guides/isaac_panda/.docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM osrf/ros:humble-desktop-jammy
+
+SHELL ["/bin/bash", "-c", "-o", "pipefail"]
+
+RUN echo "deb [trusted=yes] https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/ ./" \
+    | sudo tee /etc/apt/sources.list.d/moveit_moveit2_packages.list
+RUN echo "yaml https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/local.yaml humble" \
+    | sudo tee /etc/ros/rosdep/sources.list.d/1-moveit_moveit2_packages.list
+
+# Bring the container up to date to get the latest ROS2 humble sync on 01/27/23
+# hadolint ignore=DL3008, DL3013
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && apt-get upgrade -y && rosdep update
+
+# Install packages required to run the demo
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get install -y --no-install-recommends \
+    ros-humble-moveit \
+    ros-humble-moveit-resources
```

Comment:
If users do not have Docker buildkit set up, this will fail. So either remove the `--mount` commands here, or add the instructions to ensure they have this:

https://docs.docker.com/build/buildkit/#getting-started